Bump Meta (#1657)
Qiskit/qiskit: docs/release_notes.rst

Notable Changes
###############

*************
Qiskit 0.40.0
*************

> This release also officially deprecates the Qiskit IBMQ provider project as part of the Qiskit metapackage.

Reviewer: ```suggestion
This release officially deprecates the Qiskit IBMQ provider project as part of the Qiskit metapackage.
```

This means that in a future release ``pip install qiskit`` will no longer include ``qiskit-ibmq-provider``.
If you're currently installing or listing ``qiskit`` as a dependency to get ``qiskit-ibmq-provider`` you

> should upgrade this to explicitly list ``qiskit-ibmq-provider`` as well.

Reviewer: ```suggestion
should update this to explicitly include ``qiskit-ibmq-provider`` as well.
```

The ``qiskit-ibmq-provider`` project is still supported currently, although it has been
supersceded by the replacement package ``qiskit-ibm-provider``. For the time being both are
supported, but in the near future the ``qiskit-ibmq-provider`` package is expected to be
deprecated. This deprecation will signal a future retirement date for the package and
additionallyu means that the package but will only accepted bug fixes.

> TThis deprecation is however only for the inclusion of ``qiskit-ibmq-provider`` in the Qiskit

Reviewer 2: ```suggestion
This deprecation is however only for the inclusion of ``qiskit-ibmq-provider`` in the Qiskit
```

Reviewer 2: Sigh, I forgot the file is too big, github suggestions don't work on it.

Reviewer 2: Fixed in: https://github.com/Qiskit/qiskit/pull/1657/commits/445e8626cb29139928339b21c3015a83bf374cdd
